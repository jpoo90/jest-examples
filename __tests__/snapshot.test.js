// Added
// yarn add -D react-test-renderer
// babel-preset-react
//
// add react to babelrc
import React from 'react';
import Seasons from '../code/snapshotReact';
import renderer from 'react-test-renderer';
import { matcher, serializer } from 'jest-glamor-react';

expect.addSnapshotSerializer(serializer);
expect.extend(matcher);

it('renders correctly', () => {
  const seasonsComponent = renderer.create(<Seasons />).toJSON();
  expect(seasonsComponent).toMatchSnapshot();
});
