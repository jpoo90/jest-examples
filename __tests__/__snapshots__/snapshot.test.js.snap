// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
.css-zko0ja,
[data-css-zko0ja] {
  -ms-flex-direction: column;
  -webkit-box-pack: justify;
  -ms-flex-pack: justify;
  -webkit-box-lines: nowrap;
  -webkit-box-direction: normal;
  -webkit-box-orient: vertical;
  -webkit-justify-content: space-between;
  -webkit-flex-direction: column;
  -ms-flex-wrap: nowrap;
  -webkit-flex-wrap: nowrap;
  text-align: center;
  justify-content: space-between;
  flex-wrap: nowrap;
  flex-direction: column;
  display: -webkit-box;
  display: -moz-box;
  display: -ms-flexbox;
  display: -webkit-flex;
  display: flex;
  font-family: helvetica neue;
  font-size: 30px;
}

.css-4ua94u,
[data-css-4ua94u] {
  border-bottom: 1px dashed gray;
}

.css-lkh0sc,
[data-css-lkh0sc] {
  font-size: 40px;
  margin: 20px 0;
  color: #000;
}

.css-6nrkpz,
[data-css-6nrkpz] {
  list-style: none;
}

.css-f3khu,
[data-css-f3khu] {
  margin: 10px 0;
}

<div
  className="css-zko0ja"
>
  <div
    className="css-4ua94u"
  >
    <div
      className="css-lkh0sc"
    >
      WINTER ❄️
    </div>
    <ul>
      <li
        className="css-6nrkpz"
      >
        ⛄
      </li>
      <li
        className="css-6nrkpz"
      >
        ⛷️
      </li>
      <li
        className="css-6nrkpz"
      >
        🏂
      </li>
    </ul>
  </div>
  <div
    className="css-4ua94u"
  >
    <div
      className="css-lkh0sc"
    >
      SPRING 🌱
    </div>
    <ul>
      <li
        className="css-6nrkpz"
      >
        👩‍🌾
      </li>
    </ul>
  </div>
  <div
    className="css-4ua94u"
  >
    <div
      className="css-lkh0sc"
    >
      SUMMER ☀️
    </div>
    <div
      className="css-f3khu"
    >
      Activities coming soon ...
    </div>
  </div>
  <div
    className="css-4ua94u"
  >
    <div
      className="css-lkh0sc"
    >
      AUTUMN 🍂
    </div>
    <div
      className="css-f3khu"
    >
      Activities coming soon ...
    </div>
  </div>
</div>
`;
